### ISSUE TITLE
Version 8.7.1 not passing tests

### ISSUE THREAD

**Issue by armandomiani** [GoogleCloudPlatform/sonarqube-docker#36]
Container is building but tests on https://github.com/GoogleCloudPlatform/sonarqube-docker/blob/master/tests/functional_tests/endpoint_pgsql_test.yaml are failing. 

**Steps to reproduce**

Run the commands described in `setup` test attribute -> https://github.com/GoogleCloudPlatform/sonarqube-docker/blob/master/tests/functional_tests/endpoint_pgsql_test.yaml#L16

Use a fork from this PR branch to test: https://github.com/GoogleCloudPlatform/sonarqube-docker/pull/35

**Comment by armandomiani** [comment]
**Error from log output**

Version 8.7.1

```
2021/03/25 13:39:03  > test-0 (test REST API)
2021/03/25 13:39:03 Running command (original): [docker exec some-sonarqube-$UNIQUE-id wget -O - http://localhost:9000/api/prometheus/metrics]
2021/03/25 13:39:03 Running command (expanded): [docker exec some-sonarqube-0-0-id wget -O - http://localhost:9000/api/prometheus/metrics]
2021/03/25 13:39:04 STDERR>>--2021-03-25 13:39:03--  http://localhost:9000/api/prometheus/metrics
Resolving localhost (localhost)... 127.0.0.1, ::1
Connecting to localhost (localhost)|127.0.0.1|:9000... connected.
HTTP request sent, awaiting response... 401 

Username/Password Authentication Failed.
<<STDERR
2021/03/25 13:39:04  FAILED: Exit status 6
STDERR>>>--2021-03-25 13:39:03--  http://localhost:9000/api/prometheus/metrics
Resolving localhost (localhost)... 127.0.0.1, ::1
Connecting to localhost (localhost)|127.0.0.1|:9000... connected.
HTTP request sent, awaiting response... 401 
```

**Comment by armandomiani** [comment]
Fixed by PR #35
